src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -2,10 +2,8 @@ package pt.up.fe.ni.website.backend.controller
 
 import com.epages.restdocs.apispec.ResourceDocumentation.parameterWithName
 import com.fasterxml.jackson.databind.ObjectMapper
-import java.util.Calendar
-import java.util.Date
-import java.util.UUID
-import org.junit.jupiter.api.AfterEach
+import java.util.*
```

Comment:
I thought the linter despised wildcard imports? 👀

Reply:
I had to change my PC and had an exception for java.util. I thought the linter would pick that up though